packages/rke2-cilium/generated-changes/patch/values.yaml.patch

```diff
@@ -42,27 +53,27 @@
    port: 9962
    serviceMonitor:
      # -- Enable service monitors.
-@@ -1523,17 +1521,9 @@
+@@ -1537,17 +1537,9 @@
    # -- cilium-operator image.
    image:
      override: ~
 -    repository: "quay.io/cilium/operator"
 +    repository: "rancher/mirrored-cilium-operator"
```

Comment:
No system default registry prefix on this?

Reply:
deployment.yaml.patch includes the default registry:
```
image: "{{ template "system_default_registry" . }}{{ include "cilium.operator.image" . }}"
```